config/config.go

```diff
@@ -34,19 +35,19 @@ func LoadConfig(confName string) (*MergedConfig, error) {
 	return nil, err
```

Comment:
If you remove `var err error` this will not compile since the creation of `err` variable is only within the scope of the for loop.